benchmarks/sysbench/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+PortGroup github 1.0
+
+github.setup            akopytov sysbench 1.0.20
+revision                1
+categories              benchmarks
+platforms               darwin
+maintainers             @sainnhe \
+                        gmail.com:sainnhe
+license                 GPL-2+
+
+description             Scriptable database and system performance benchmark
+
+long_description \
+    sysbench is a scriptable multi-threaded benchmark tool based on LuaJIT. It is \
+    most frequently used for database benchmarks, but can also be used to create \
+    arbitrarily complex workloads that do not involve a database server.
+
+homepage                https://github.com/akopytov/sysbench
+
+checksums               rmd160  ee4e41c3a5f5f18c80787bc608f84faf33f1c989 \
+                        sha256  bab986be195206acdaa6609c7cf4556d2e2e222806cca44569683a6e04e00ac9 \
+                        size    1509764
+
+depends_build-append    port:pkgconfig \
+                        port:automake
+
+depends_lib             port:libtool \
+                        path:lib/libssl.dylib:openssl
```

Comment:
How does this port use openssl? It doesn't appear to link with the openssl libraries...

Reply:
Yeah, the executable is not linked to openssl, but the [build instruction](https://github.com/akopytov/sysbench#macos-1) said you need to install openssl. I have no idea what's going on.

Reply:
@ryandesign Do you think openssl should be removed?

Reply:
Never trust upstream instructions. 😝 If it uses openssl, declare a dependency on it. If it doesn't, don't.

Reply:
Thanks! I've removed openssl from dependency.